### ISSUE TITLE
full of bugs

### ISSUE THREAD

**Issue by JiangNanMax** [lqxhub/go_epoll#1]
1) incorrect connection-handle
2) concurrent-unsafe

**Comment by lqxhub** [comment]
你说的第一个, 错误的处理, 还请你指正, 我好改正; 第二个 并发安全问题, 是指并发读写map的问题吗, 但是现在对map的操作都是在同一个协程做的, 应该不会有并发安全问题

**Comment by LSYGOD** [comment]
TestClients不是并发的

**Comment by lqxhub** [comment]
@JiangNanMax @LSYGOD 确实, 我今天看了一下, 主协程有个异步的 `HandlerEpoll()` 没有注意到, `Accept()`和 `HandlerEpoll()` 是两个协程,这两个协程都会操作 conn 这个map, 是有问题的, 谢谢提醒
